inference-engine/tests/functional/plugin/cpu/transformations/move_eltwise_up_data_movement.cpp

```diff
@@ -0,0 +1,353 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <transformations/init_node_info.hpp>
+#include <ngraph/pass/manager.hpp>
+#include "ngraph_functions/builders.hpp"
+#include "lpt_ngraph_functions/common/builders.hpp"
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "ngraph_transformations/move_eltwise_up_data_movement.hpp"
+
+using namespace testing;
+using namespace ngraph;
+
+namespace SubgraphTestsDefinitions {
```

Comment:
This is not a subgraph test, this is a transformation test. So the namespace name `SubgraphTestsDefinitions` looks rather weird. I am not sure that we need this namespace at all.